### DISCUSSION TITLE
Simplify the process and the nomination of "become a member"

### DISCUSSION THREAD

**Discussion by STRRL** [chaos-mesh/chaos-mesh#3433] [General]
As [the governance documentation](https://github.com/chaos-mesh/chaos-mesh/blob/master/GOVERNANCE.md) says, one contributor becoming the member requires the committer and maintainer to open a PR as nomination and then vote.

But there is no file to record the members, so no file changes, no PR creates.

I suggest that we just simplify the process as:

- any committer and maintainer as a sponsor could **directly invite** the active contributor to become a member
- then the sponsor would announce this activity on the [slack channel #project-chaos-mesh](https://cloud-native.slack.com/archives/C0193VAV272)

**Comment by STRRL** [comment]
PTAL @dcalvin @luzizhuo

**Reply by g1eny0ung** [reply]
Speaking of which, I also suggest that we also have the right to remove people from committers who haven't contributed for a long time (This is at least good for the robot to automatically select people for review).

**Reply by STRRL** [reply]
+1.

I think we should set the automatic rules for "Retired/Emeritus" Maintainers and Committers, for who have built remarkable contributions for Chaos Mesh, but are not available for taking participate in the development/community activity anymore.

**Reply by STRRL** [reply]
PTAL @dcalvin @cwen0 @luzizhuo

**Reply by g1eny0ung** [reply]
> +1.
> 
> I think we should set the automatic rules for "Retired/Emeritus" Maintainers and Committers, for who have built remarkable contributions for Chaos Mesh, but are not available for taking participate in the development/community activity anymore.

+1.

We still appreciate their contributions and they can continue to exercise their entitlements as contributors to the project. This is also a consideration for the development of the project, the maintainer/committers are always changing (Maybe one day I will retire too 😆).

**Reply by dcalvin** [reply]
+1. This makes sense. And it is necessary to clarify this with the community.

**Reply by luzizhuo** [reply]
+1. I think how to define "Retired/Emeritus Maintainers and Committers" is a matter for discussion. Automatic or manual? By the contributions on GitHub or overall contributions to the community? What if a Maintainer/Committer temporarily leaves the community and returns later?

**Reply by g1eny0ung** [reply]
> +1. I think how to define "Retired/Emeritus Maintainers and Committers" is a matter for discussion. Automatic or manual? By the contributions on GitHub or overall contributions to the community? What if a Maintainer/Committer temporarily leaves the community and returns later?

@STRRL @dcalvin @cwen0 @luzizhuo

One of my hypotheses is to take a **six-month** cycle, and if some maintainers/committers aren't active within six months, then any existing maintainers/committers can raise an issue to vote on whether to change him to contributors.

If he ever comes back, they will still be contributors, and no one will stop them from contributing or discussing any topic. 🙌🏽 He/She can remain active and become a committer again.

I think from now there were no "Retired/Emeritus Maintainers and Committers", they should all be contributors, just they can exist in the Chaos Mesh GitHub Org. It no longer makes sense to distinguish these titles. **There is nearly no difference between maintainers/committers and contributors, all we do the same thing, communicate and develop.**

Just the difference in the time we pay.

**Comment by cwen0** [comment]
LGTM

**Comment by luzizhuo** [comment]
LGTM

**Comment by Andrewmatilde** [comment]
LGTM

**Comment by STRRL** [comment]
I would open a PR to update the governance doc.

**Reply by STRRL** [reply]
https://github.com/chaos-mesh/chaos-mesh/pull/3450
